pkg/config/proto/cfg.pb.go

```diff
@@ -826,55 +826,62 @@ func init() {
-	0xef, 0xdc, 0x2c, 0x48, 0xc9, 0xed, 0xdc, 0x9b, 0x18, 0x7b, 0xcf, 0xbd, 0xb7, 0x75, 0x05, 0x65,
-	0xc5, 0xd6, 0xa1, 0x74, 0x47, 0xfc, 0x91, 0xca, 0xdc, 0x04, 0x1d, 0x8d, 0x50, 0x20, 0x30, 0x53,
-	0x2d, 0xfd, 0x05, 0xca, 0x01, 0x62, 0xc8, 0x8b, 0xda, 0xa2, 0xb5, 0x6b, 0xbd, 0xef, 0xec, 0x78,
-	0xf1, 0xed, 0x64, 0xf1, 0xed, 0x5b, 0xd9, 0x0d, 0xeb, 0x0d, 0xe8, 0xfd, 0x98, 0x8c, 0x24, 0x7d,
-	0xe4, 0x61, 0x25, 0x99, 0x24, 0x28, 0x24, 0xbf, 0x88, 0x17, 0xb8, 0x66, 0xf1, 0x79, 0xea, 0x9f,
-	0x4a, 0x50, 0xff, 0xd7, 0xa5, 0x43, 0xe4, 0xaa, 0x6d, 0x48, 0x0f, 0x3f, 0xce, 0xb3, 0x07, 0x15,
-	0x55, 0x7a, 0xd2, 0xb5, 0x9f, 0x96, 0xf5, 0x41, 0xd5, 0xf9, 0x27, 0x54, 0x3d, 0xe4, 0x93, 0x31,
-	0xe6, 0x22, 0x69, 0x5d, 0x7b, 0x19, 0x22, 0x04, 0x23, 0xc3, 0x50, 0xe0, 0x4b, 0x05, 0x18, 0xc7,
-	0x50, 0x72, 0xe9, 0x24, 0xd9, 0x96, 0xfd, 0x25, 0x5c, 0xea, 0x11, 0x5c, 0xd0, 0xc9, 0x59, 0xf4,
-	0xec, 0x4f, 0xef, 0x8f, 0x8c, 0x13, 0xd0, 0xe3, 0x67, 0x9d, 0xac, 0x4e, 0x6f, 0xb5, 0xcc, 0xa5,
-	0x04, 0x52, 0x22, 0xaf, 0x61, 0x33, 0xfb, 0xce, 0xb9, 0x59, 0x96, 0x82, 0x7f, 0x3d, 0x41, 0x30,
-	0x81, 0x6f, 0x14, 0x9b, 0xd2, 0x3e, 0x07, 0xb8, 0xf7, 0x00, 0x6e, 0xea, 0x52, 0xf2, 0x60, 0xb5,
-	0xe4, 0x55, 0xc2, 0xa4, 0xa4, 0xfa, 0x50, 0x96, 0xbe, 0xc0, 0xcd, 0x8a, 0x94, 0xe9, 0xae, 0x96,
-	0xb9, 0x8e, 0xe2, 0x53, 0x12, 0x7b, 0x50, 0x79, 0x8b, 0xfc, 0x91, 0x1b, 0x4d, 0xb8, 0x9a, 0x3b,
-	0xe1, 0xff, 0xe2, 0x30, 0xf9, 0xda, 0x3e, 0x17, 0x60, 0x23, 0x3b, 0xbe, 0xec, 0xfe, 0x24, 0x5b,
-	0x1b, 0xef, 0xe9, 0x05, 0x00, 0x4a, 0xa0, 0x64, 0x39, 0x0e, 0x9f, 0xba, 0x1c, 0xf3, 0x13, 0x2e,
-	0x47, 0xde, 0xc2, 0xd0, 0xb8, 0x3f, 0x3a, 0xf7, 0xc7, 0xd4, 0xd8, 0x84, 0x5a, 0x72, 0xd1, 0x79,
-	0x05, 0xff, 0x00, 0xcc, 0x7d, 0x50, 0xd6, 0xf1, 0x4d, 0x6f, 0x77, 0x75, 0x9b, 0x5e, 0x45, 0xcc,
-	0x8b, 0x59, 0x80, 0x5b, 0x04, 0x9a, 0x0b, 0x99, 0x8d, 0x1a, 0x14, 0xef, 0xf0, 0x4c, 0x25, 0x38,
-	0x83, 0x35, 0x99, 0x40, 0x6a, 0xd7, 0x7a, 0x7f, 0x3c, 0xff, 0x3e, 0x51, 0xf1, 0x47, 0x85, 0x43,
-	0xcd, 0xfa, 0x16, 0x8a, 0x2f, 0x19, 0x31, 0x1a, 0x89, 0xa2, 0x4c, 0x60, 0xb5, 0xa0, 0x7a, 0x1e,
-	0xf4, 0x47, 0x23, 0x86, 0x39, 0x7f, 0xf8, 0xad, 0x6e, 0x99, 0xa0, 0x9f, 0xfa, 0xfc, 0x7f, 0xe4,
-	0xe1, 0x45, 0x6a, 0x1b, 0xea, 0x67, 0x1e, 0x22, 0xee, 0xa3, 0x60, 0xd5, 0x12, 0x50, 0xca, 0xf1,
-	0x49, 0xe9, 0xbf, 0x2b, 0x7d, 0x52, 0x46, 0x19, 0xbf, 0x3e, 0xf4, 0xc9, 0x3c, 0xa3, 0xb6, 0x7e,
-	0x83, 0xb2, 0xa2, 0x9a, 0xa0, 0xab, 0xb5, 0x53, 0xbb, 0xb1, 0x01, 0x55, 0xe2, 0x73, 0x81, 0x7c,
-	0x47, 0x49, 0x55, 0xad, 0x01, 0xd4, 0xd2, 0x7f, 0x0c, 0x5b, 0xd0, 0x48, 0x22, 0x06, 0x29, 0x2b,
-	0x5c, 0x87, 0x8a, 0xc0, 0x5e, 0xe0, 0x22, 0x81, 0x33, 0xf6, 0x5a, 0xcc, 0x37, 0xc0, 0x6b, 0xd0,
-	0xd5, 0x52, 0x2f, 0xd8, 0x6b, 0x8e, 0x63, 0xe7, 0x4b, 0x0e, 0xcb, 0xf2, 0xe0, 0xf7, 0xaf, 0x01,
-	0x00, 0x00, 0xff, 0xff, 0xe0, 0xd0, 0x33, 0xe1, 0xfd, 0x08, 0x00, 0x00,
+	// 898 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0x94, 0x55, 0x5b, 0x8f, 0xdb, 0x44,
```

Comment:
This is temporary, it will be replace by bazel build rule. It is generated by using the script https://github.com/istio/mixer/blob/master/pkg/config/proto/build_cfg.sh. We have plans to replace the script with a build rule.
thanks for approving.